ccc/github.py

```diff
@@ -63,6 +67,31 @@ def github_api_ctor(
         raise ValueError('failed to parse url: ' + str(github_url))
 
     session = github3.session.GitHubSession()
+    original_request = session.request
+
+    def intercepted_request(*args, **kwargs):
+        req = original_request(*args, **kwargs)
+
+        try:
+            method, url = args
+            es_client.store_document(
+                index='github_request',
+                body={
+                    'method': method,
+                    'url': url,
+                    'data': kwargs,
+                    'creation_date': datetime.datetime.now().isoformat(),
+                },
+            )
+        except:
+            # dont raise due to logging
+            logger.warning('unable to log session requests to ES')
```

Comment:
please do not abbreviate elasticsearch to just ES - this will make the log output very hard to understand

also: please _do_ state that we were unable to log a Github-API-Request! just calling this a "session request" if meaningless w/o knowing the code. Please consider there might be readers of the warnings that do not know our code-base in detail, and just see the warning-messages.